flink-connector/src/main/java/io/delta/flink/sink/logging/Logging.java

```diff
@@ -1,4 +1,4 @@
-package io.delta.flink.sink.internal.logging;
+package io.delta.flink.sink.logging;
```

Comment:
why is this needed?
Logging should not be exposed

Reply:
logging is used also by classes like `DeltaSink` or `DeltaSinkBuilder` so its methods need to be visible to them. 
Even if we create `AbstractLogging` interface that could be used by those non-internal classes then this interface will still need to have its methods defined (otherwise those non-internal classes will not compile).  
In the effect - we will still have some sort of logging interface in the Java docs with methods' names so seems like a better solution not to create an abstract version of it.